generate-deb.sh

```diff
@@ -432,15 +432,15 @@ Source: dropbox
 Section: gnome
 Priority: optional
 Maintainer: Dropbox <support@dropbox.com>
-Build-Depends: cdbs, debhelper (>= 9), build-essential, libnautilus-extension-dev (>= 3.10.1), libglib2.0-dev (>= 2.40), python-gtk2 (>= 2.24), python-docutils
+Build-Depends: cdbs, debhelper (>= 9), build-essential, libnautilus-extension-dev (>= 3.10.1), libglib2.0-dev (>= 2.40), python-gi (>= 3.20), python-docutils
 Standards-Version: 3.9.4.0
 
 Package: dropbox
 Replaces: nautilus-dropbox
 Breaks: nautilus-dropbox
 Provides: nautilus-dropbox
 Architecture: any
-Depends: procps, python-gtk2 (>= 2.24), python (>= 2.7), \${python:Depends}, \${misc:Depends}, libatk1.0-0 (>= 2.10), libc6 (>= 2.19), libcairo2 (>= 1.13), libglib2.0-0 (>= 2.40), libgtk2.0-0 (>= 2.24.23), libpango1.0-0 (>= 1.36.3), lsb-release
+Depends: procps, python-gi (>= 3.20), python (>= 2.7), \${python:Depends}, \${misc:Depends}, libatk1.0-0 (>= 2.10), libc6 (>= 2.19), libcairo2 (>= 1.13), libglib2.0-0 (>= 2.40), libgtk-3-dev (>= 3.0), libpango1.0-0 (>= 1.36.3), lsb-release
```

Comment:
I think the minimum python-gi version should be 3.12, since Trusty ships with 3.12.0 (https://packages.ubuntu.com/search?keywords=python-gi&searchon=names&suite=trusty&section=all) and Debian Jessie ships with 3.14 (search for python-gi here: https://packages.debian.org/jessie/allpackages?format=txt.gz)

Also, I believe you need to depend on `libgtk3-0`, not `libgtk3-dev` (those include the development headers, which our users don't need). The minimum version should be `3.10`, since that's the minimum version shipped by Trusty and Jessie (Trusty ships 3.10, Jessie ships 3.14).

Reply:
Looking at https://github.com/dropbox/nautilus-dropbox/pull/57, I think you'd need more runtime deps:
```
gir1.2-gdkpixbuf-2.0,
gir1.2-glib-2.0,
gir1.2-gtk-3.0,
gir1.2-pango-1.0,
```

Reply:
Thanks for letting me know! It looks like some of them are not needed to be in deps since they are in the dependency chain of python-gi, but I'll put all of them in the runtime dependencies to make sure.

Reply:
Hmm... in Ubuntu 18.04 and Debian Testing `python-gi` has only one GI package in its dependencies - `gir1.2-glib-2.0`.